axelrod/strategies/lookerup.py

```diff
@@ -232,10 +233,12 @@ class LookerUp(Player):
 
     The dictionary must contain all possible permutations of C's and D's.
 
-    LookerUp can also be instantiated with pattern=str/tuple of actions, and
-    parameters=Plays(self_plays=player_depth: int,
-                     op_plays=op_depth: int,
-                     op_openings=op_openings_depth: int).
+    | LookerUp can also be instantiated with pattern=str/tuple of actions, and
```

Comment:
I hadn't seen the `|` in rst before. I'm not sure it looks like it should though. Here's what the docs build looks like on this branch:

![screenshot from 2017-04-13 09-46-20](https://cloud.githubusercontent.com/assets/2131546/24997347/3b69a5ce-202e-11e7-9a5e-fa9b2f2ef534.png)

Is that what you were going for?

This tweak:

```
  235                                                                                                                                                 
~ 236     LookerUp can also be instantiated with :code:`pattern=str/tuple` of actions                                                                 
~ 237     , and::                                                                                                                                     
~ 238                                                                                                                                                 
~ 239         parameters=Plays(                                                                                                                       
~ 240             self_plays=player_depth: int,                                                                                                       
+ 241             op_plays=op_depth: int,                                                                                                             
+ 242             op_openings=op_openings_depth: int).                                                                                                
  243                                                                                                                                                 
  244     It will create keys of len=2 ** (sum(parameters)) and map the pattern to                                                                    
  245     the keys.   
```
looks like:

![screenshot from 2017-04-13 09-50-35](https://cloud.githubusercontent.com/assets/2131546/24997484/ba92daa0-202e-11e7-99fe-9e53c1092e0d.png)

> while i think of it, can i go ahead and delete all the commented-out original data?

Sounds good to me. :+1: